app/views/projects/samples/attachments/_table.html.erb

```diff
@@ -10,6 +10,7 @@
         <% if allowed_to?(:update_sample?, @project) %>
           <th aria-hidden="true" class="px-6 py-3"></th>
         <% end %>
+        <th scope="col" class="px-6 py-3"><%= t("projects.samples.show.table_header.puid") %></th>
         <th scope="col" class="px-6 py-3"><%= t("projects.samples.show.table_header.filename") %></th>
         <th scope="col" class="px-6 py-3"><%= t("projects.samples.show.table_header.format") %></th>
```

Comment:
Maybe a question for @joshsadam as well. Just wondering about consistency in styling, if we wanted to match the darkened background of the PUID column on attachments like it exists for samples. As well as if we want the PUID clickable to download like the name.

Reply:
1.  The dark background on the samples page is just there to handle the horizontal scrolling and to create a difference.  We could update all tables to have their primary column dark, but I don't think that is part of this PR.
2.  Clicking the name to download is something that we should not have had.  It needs to be moved (not in this PR) to where the delete button is.  The delete button should not be an actual button, but just a text link like on the [Flobite page](https://flowbite.com/docs/components/tables/#striped-rows).